lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2873,6 +2874,11 @@ def __init__(self, **kwargs):
             option_minimal_versions_ignore_params=self.__NON_CONTAINER_PROPERTY_OPTIONS,
             **kwargs
         )
+
+        self.inspect_source = 'Config'
+        if self.docker_api_version < LooseVersion('1.21'):
+            self.inspect_source = 'ContainerConfig'
```

Comment:
I would rename the property to `image_inspect_source`, since this is not about container inspects.